src/Symfony/Component/Routing/Generator/Dumper/CompiledUrlGeneratorDumper.php

```diff
@@ -35,12 +37,69 @@ public function getCompiledRoutes(): array
                 $compiledRoute->getTokens(),
                 $compiledRoute->getHostTokens(),
                 $route->getSchemes(),
+                [],
             ];
         }
 
         return $compiledRoutes;
     }
 
+    public function getCompiledAliases(): array
+    {
+        $routes = $this->getRoutes();
+        $compiledAliases = [];
+        foreach ($routes->getAliases() as $name => $alias) {
+            $deprecations = [];
+
+            if ($alias->isDeprecated()) {
+                $deprecation = $alias->getDeprecation($name);
+
+                $deprecations[] = $deprecation;
+
+                trigger_deprecation($deprecation['package'], $deprecation['version'], $deprecation['message']);
```

Comment:
doesn't this mean there is no way to work around the deprecation? aka a deprecation is always raised when an alias is deprecated, even if it's never used?